workspaces/jupyterlab-python/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM jupyter/minimal-notebook:2023-10-02
+COPY jupyter_notebook_config.py /etc/jupyter/
+
+ENV XDG_DATA_HOME=/tmp/local/share
+ENV XDG_CACHE_HOME=/tmp/cache
+ENV XDG_CONFIG_HOME=/tmp/config
+ENV JUPYTER_RUNTIME_DIR /tmp/runtime
+ENV JUPYTER_ALLOW_INSECURE_WRITES true
+ENV JUPYTER_CONFIG_DIR=/tmp/jupyter_config
+ENV NPM_CONFIG_CACHE=/tmp/npm
+ENV NO_UPDATE_NOTIFIER=true
+ENV IPYTHONDIR=/tmp/ipython
+ENV YARN_CACHE_FOLDER=/tmp/yarn_cache
```

Comment:
This was copied without modification from the old `workspaces/jupyterlab/Dockerfile`. I didn't test which of these were actually necessary.